library/grpc/v1.49.0/base/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# syntax=docker/dockerfile:1.4
+ARG DOCKER_ORG=bufbuild
+ARG BASE_BUILD_VERSION=latest
+FROM ${DOCKER_ORG}/plugins-grpc-base-build:${BASE_BUILD_VERSION}
+
+RUN git clone --depth 1 --branch v1.48.1 --recursive https://github.com/grpc/grpc
```

Comment:
It's too bad these are hard-coded, I wonder if we could search/replace the version and bump in both the `buf.plugin.yaml` file _and_ the Dockerfile. Wdyt?